src/SprykerEco/Yves/Adyen/Plugin/Payment/CreditCardPaymentMapperPlugin.php

```diff
@@ -15,6 +15,8 @@
 class CreditCardPaymentMapperPlugin implements AdyenPaymentMapperPluginInterface
 {
     /**
+     * {@inheritDoc}
+     *
```

Comment:
* Add an `@api` mark to a method.
* Do the same for a similar places.

Reply:
Enable GitHub actions by a copy them from a repository: 
* https://github.com/spryker-eco/avalara-tax-product-option/pull/3/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03f